client/cmd/dexc/main.go

```diff
@@ -75,7 +75,7 @@ func main() {
 	// At least one of --rpc or --web must be specified.
```

Comment:
Just noticed that this comment is outdated.

Reply:
Right, I'll change it.